Development/nmos/control_protocol_methods.cpp

```diff
@@ -335,7 +335,7 @@ namespace nmos
                     // do notification that the specified property has changed
                     if (property_changed)
                     {
-                        property_changed(resource, nmos::fields::nc::name(property), index);
+                        property_changed(resource, nmos::fields::nc::name(property), -1);
```

Comment:
```suggestion
                        property_changed(resource, nmos::fields::nc::name(property), -2);
```

Reply:
- Currently, we use `-1` for non-sequence changed; maybe we can use `-2` to identify this as a deleted sequence item changed. 
- We must update the comment for the `control_protocol_property_changed_handler` in control_protocol_handlers.h.
- Also, update the example, `make_node_implementation_control_protocol_property_changed_handler` in node_implementation.cpp